docs/pages/blog/mui-core-v5.md

```diff
@@ -0,0 +1,678 @@
+- use [`tss-react`](https://github.com/garronej/tss-react) - its API is similar to JSS `makeStyles` but is powered by emotion, which minimizes the bundle duplication with the core components.
+
+You can find more information for this on the [Migrate from JSS](/guides/migration-v4/#migrate-from-jss) section of the migration guide.
+
+### Change to the supported platforms
+
+This breaking change is an opportunity to drop the support of legacy upstream dependencies.
+
+- We have updated the minimum supported TypeScript version from 3.2 to **3.5**.
+  This aims to match the policy of [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped), with versions of TypeScript that are less than two years old.
+- We have updated the minimum supported node.js version from 8.0 to **12.17**.
+  This aims to match the [LTS versions that are in maintenance](https://github.com/nodejs/Release#release-schedule) mode.
+- We have updated the minimum supported React version from 16.8 to **17.0**.
+  The breaking changes released between the two versions are [very limited](https://reactjs.org/blog/2020/10/20/react-v17.html).
+- We have updated the supported browsers.
+  - IE: **partial**. We have kept the logic added in the past to support IE 11,
+    however, we have stopped actively working on it. We can't guarantee that it works correctly. It's discontinued.
+  - Edge: from 14 to **91**. The minimum version based on Chromium.
+  - Firefox: from 52 to **78**.
+  - Chrome: from 49 to **90**. We have [assumed](https://developers.google.com/search/blog/2019/05/the-new-evergreen-googlebot) that Googlebot is always using the latest version of Chrome.
+  - Safari: from 10 to **12.5**
+
+These changes have allowed us to save [6 kB gzipped](https://github.com/mui-org/material-ui/pull/22814#issuecomment-700995216) on the `@mui/material` package.
+
+## Design kits
+
+We now have the MUI components available in the three main design tools out there: Figma, Sketch, and Adobe XD. If you work together with a team of designers, having them using the UI kits makes for easier communication as you'll be speaking the same language around the components. It'll also help designers to understand how the library is structured, especially theming features and each component prop, enabling a much faster development.
+
+## What's next?
+
+Looking back, we are glad to say that, together, we have accomplished most of the objectives we defined in the [v4 release blog post](/blog/material-ui-v4-is-out/#whats-next).
+We are also proud to share that currently we are at around 25% of market share in the React community. We aim to grow till around 50% by 2025.
+This is an ambitious goal, but not impossible. We have managed to make this effort sustainable in the past, for example by introducing the [premium themes store](/blog/material-ui-v4-is-out/#premium-themes-store).
+In the following sections we are going to share our public roadmaps and the high level areas we plan to focus on.
```

Comment:
```suggestion
In the following sections, we share our public roadmaps and the high-level areas we plan to focus on.
```

(or "are sharing"? not sure about this one)

Reply:
cc @mbrookes ?